hw/top_earlgrey/syn/chip_earlgrey_asic_check_only.sdc

```diff
@@ -0,0 +1,53 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# These SDC's are not used for synthesis, since they collide with the current
+# set_clock_groups -async constraints. Instead, they are used to time the design
+# after P&R in order to make sure that the CDC crossings are all within spec.
+
+#####################
+# main clock        #
+#####################
+# target is 100MHz, overconstrain by factor
+set MAIN_TCK_TARGET_PERIOD  10
+set MAIN_TCK_FACTOR 0.85
+set MAIN_TCK_PERIOD [expr $MAIN_TCK_TARGET_PERIOD*$MAIN_TCK_FACTOR] ;# over constraining
+
+#####################
+# CDC               #
+#####################
+
+# first, set maximum delay from any FF to a 2 FF synchronizer to 0.9 x the shortest clock period in the system.
+# note that this implicitly checks all alert handler differential channels, since these use 2FF synchronizers as well.
+set MAX_DELAY [expr $MAIN_TCK_PERIOD * 0.9]
+set_max_delay $MAX_DELAY -to [get_pins -hierarchical -filter "full_name =~ *u_sync_1/gen_*u_impl*/gen_flops?0?*?u_size_only_reg/D"]
+
+# for prim_sync_reqack_data instances
+# these are all through paths with data just being routed throgh data_i and data_o with no FF in between.
+set_max_delay $MAX_DELAY -through [get_pins -hierarchical *u_prim_sync_reqack_data/data_i]
+set_max_delay $MAX_DELAY -through [get_pins -hierarchical *u_otp_key_req_sync/data_i]
+
+# this one has a data reg
+set_max_delay $MAX_DELAY -to [get_pins -hierarchical -filter "full_name =~ *u_prim_sync_reqack_data_in/gen_data_reg*data_q*/data_in"]
```

Comment:
This one returns the Error: Nothing matched for pins.
@arnonsha

Reply:
@msfschaffner 
did you have the chance to check these constraints in DC on the real netlist ?

Reply:
I checked this on the DDC that you provided.

Could it be that there are unmapped registers in that DDC that have other pin names?

Does this constraint work if the pin name `data_in` is replaced by `D`?